reduction/Imaging_instructions.txt

```diff
@@ -72,3 +72,40 @@ of the important scripts (i.e., this directory:
 https://github.com/ALMA-IMF/reduction/tree/master/reduction).
 
 When you create mask region files, they belong in $ALMAIMF_ROOTDIR/clean_regions/.
+
+
+Additional tricks of cleaning in selfcal
+=============================================
+Be conservative in making the first image before the first iteration of the selfcal.
+That is, clean relatively relatively shallow probably by setting the threshold to be
+4 times rms or a bit greater. Otherwise, some artefacts may be introduced into the final
+image. In the later iterations (e.g., from iter2, 3, ...,) of selfcal, the threshold can be
+constrained down to 2 times rms.
+
+Scales in band3 could be set larger than those in the band 6, which is demonstrated for the 
+source G353.41, for example, scales = [0, 3, 9, 27] in band3 but scales = [0, 3, 9] in band6. 
+Note that the scales are ought to be in a reasonable range, i.e., smaller than
+or comparable to the size the largest mask, otherwise, the final image could be distorted.
+
+In each selfcal, the threshold and the region mask could be changeable due to the rms varying
+with each selfcal. So, it is recommend to change them following your previous criteria 
+on the threshold and region mask. 
+
+The mask can be made with different criteria for dense and
+diffuse regions. For example, the dense regions can be masked out with a contour of 4*rms, but
+the diffuse regions can be done with 3*rms. Checking the residual image from the right previous selfcal
+is always useful to make a good mask for a good cleaning that could result in a more flatter
+residual image in the next iteration of selfcal.
```

Comment:
At least the first half of this paragraph is already made clear in https://github.com/ALMA-IMF/notebooks/blob/master/SelfCal_Instructions_Examination.ipynb .  But, pointing people to look at the residuals to decide where to put additional regions is a good idea